protomoney/math.go

```diff
@@ -0,0 +1,81 @@
+package protomoney
+
+import (
+	"google.golang.org/genproto/googleapis/type/money"
+)
+
+// Abs returns the absolute value of this amount.
+//
+// That is, if m is negative, it returns its inverse (a positive magnitude),
+// otherwise it returns m unchanged.
+func Abs(m *money.Money) *money.Money {
+	if IsPositive(m) {
+		return m
+	}
+
+	return &money.Money{
+		CurrencyCode: m.CurrencyCode,
+		Units:        -m.Units,
+		Nanos:        -m.Nanos,
+	}
+}
+
+// Neg returns -m.
```

Comment:
Should this function have more of a description like the others?

Reply:
Something like `// Neg returns the negated value of this amount.`

Reply:
Possibly, all of the common math operators are described this way, also on the `Amount` type, happy to expand them but it's probably a slightly larger change.

Reply:
All good. It just looked different to all the others, but I guess those were expanded to mention the panic on currency mismatch.